### ISSUE TITLE
Option to have CodeGlance on the left of scrollbar

### ISSUE THREAD

**Issue by erusev** [Vektah/CodeGlance#224]
The scrollbar shows on the left of the minimap:

![image](https://user-images.githubusercontent.com/184170/50967457-e5137c00-14e0-11e9-8238-2ec83a42fbda.png)

But for some people it might make more sense for the scrollbar to stay at its original position, right next to the right border of the editor, and for the minimap to show on the left of the scrollbar, as in VSC:

![image](https://user-images.githubusercontent.com/184170/50967602-3de31480-14e1-11e9-93cb-2fd1ada21d45.png)

**Comment by hwooo** [comment]
I agree. It would be best if the scrollbar is fixed to right of the minimap.

**Comment by mustafaozhan** [comment]
Agreed 👍
